.changeset/small-ads-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-components': patch
+---
+
+Added linting configuration to ensure all Stencil components conform to the [Stencil style guide](https://stenciljs.com/docs/style-guide).
```

Comment:
For internal changes, it's not necessary to write a changeset, because we're not changing anything on the components and therefore it's less relevant for devs (https://github.com/swisspost/design-system/blob/main/CONTRIBUTING.md#versioning)

Reply:
I just removed the changeset.